contracts/key_store/src/lib.rs

```diff
@@ -53,7 +53,14 @@ mod key_store {
         }
 
         #[ink(message)]
-        pub fn set_executor(&mut self, executor: AccountId) -> Result<()> {
+        pub fn renounce_admin(&mut self, new_admin: AccountId) -> Result<()> {
```

Comment:
```suggestion
        pub fn transfer_admin(&mut self, new_admin: AccountId) -> Result<()> {
```

This is a transfer of the ownership, not discarding.

Reply:
Renamed to `transfer_ownership`